.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,38 @@
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```

Markdownlint (0.37.0)

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)